kernel/rros/net/device.rs

```diff
@@ -37,9 +37,10 @@ struct ListThreadSafeWrapper(pub List<Box<NetDevice>>);
 unsafe impl Sync for ListThreadSafeWrapper {}
 unsafe impl Send for ListThreadSafeWrapper {}
 
-init_static_sync! {
-    static ACTIVE_PORT_LIST : SpinLock<ListThreadSafeWrapper> = ListThreadSafeWrapper(List::new());
-}
+// init_static_sync! {
+//     static ACTIVE_PORT_LIST : SpinLock<ListThreadSafeWrapper> = ListThreadSafeWrapper(List::new());
+// }
+static ACTIVE_PORT_LIST : Pin<Box<SpinLock<ListThreadSafeWrapper>>> = Box::pin_init(new_spinlock!(ListThreadSafeWrapper(List::new()))).unwrap();
 
```

Comment:
cc @shannmu

Reply:
It is good for me for now. `Box::pin_init` returns a `Result` and using `unwrap` here is a temporary solution. I believe we need to address this problem later.

Reply:
ok